charts/opentelemetry-collector/templates/service.yaml

```
@@ -5,9 +5,9 @@ metadata:
   name: {{ include "opentelemetry-collector.fullname" . }}
   labels:
     {{- include "opentelemetry-collector.labels" . | nindent 4 }}
-  {{- with .Values.service.annotations }}
+  {{- if .Values.service.annotations }}
   annotations:
-    {{ toYaml . | trim | indent 4 }}
+{{ toYaml .Values.service.annotations | trim | indent 4 }}
```

```suggestion
    {{ toYaml .Values.service.annotations | nindent 4 }}
```

nit: little more readable imho